wallet/db.c

```diff
@@ -328,6 +328,8 @@ char *dbmigrations[] = {
     "INSERT OR IGNORE INTO blocks (height) VALUES ((SELECT MIN(first_blocknum) FROM channels));",
     "DELETE FROM blocks WHERE height IS NULL;",
     /* -- End of  PR #1398 -- */
+    "ALTER TABLE invoices ADD description BLOB;",
+    "ALTER TABLE payments ADD description BLOB;",
```

Comment:
TEXT?  We really can't handle anything but UTF-8 here.  To be SQL compliant we could use VARCHAR(639), but we're not SQL compliant elsewhere anyway...

Reply:
Good point, `VARCHAR(639)` would also not be supported by MySQL xD